src/api/admin.ts

```diff
@@ -42,6 +42,20 @@ export interface EventInput {
   host: string;
 }
 
+export interface WorldScheduleEvent {
+  name: string;
+  description: string;
+  startDate: string;
+  startTime: string;
+  durationHours: number;
+  durationMinutes?: number;
+  host: string;
+  space?: {
+    id?: string;
+    slug?: string;
+  };
```

Comment:
do we need `slug` here? can't we just save `spaceId`?

Reply:
`slug` is not replaced with `name` we need the name to show in the dropdown but we need the `id` to be passed to the function and update the venue.